src/openshift/openshiftItem.ts

```diff
@@ -32,16 +32,29 @@ export abstract class OpenShiftItem {
         return new Promise((res) => setTimeout(res, timeout));
     }
 
+    static validateName(data: Array<OpenShiftObject>, value: string) {
```

Comment:
Can you at least name this to be indicative of what it checks for? 'validateName' is a little too broad for what it does. Something in line with the other validators could do.